### DISCUSSION TITLE
CUPS - Common Unix Printing System

### DISCUSSION THREAD

**Discussion by vrytired** [selfhst/icons#119] [New Icon Request]
Original Project: https://www.cups.org/
Currently Maintained Fork: https://openprinting.github.io/cups/

Wikipedia has an existing logo: https://en.wikipedia.org/wiki/CUPS

**Comment by shollyethan** [comment]
> Original Project: https://www.cups.org/ Currently Maintained Fork: https://openprinting.github.io/cups/
> 
> Wikipedia has an existing logo: https://en.wikipedia.org/wiki/CUPS

The fork you provided seems to be using the logo below - would you prefer this or the logo from Wikipedia?

https://github.com/OpenPrinting/cups/blob/master/desktop/cups.svg

**Comment by vrytired** [comment]
No preference, whatever you think is best.

**Comment by shollyethan** [comment]
I've added both the official CUPS icon as well as the OpenPrinting CUPS icon - thanks!
